pheps/phep-0001.md

```diff
@@ -0,0 +1,366 @@
+
+# PHEP Motivation
+<a name="phep-motivation"></a>
+The [PyHC standards](https://doi.org/10.5281/zenodo.2529131) have not been updated since their initial approval in 2018, and there is no established process for updating them. PyHC discussions often involve what packages "should" do, or "should be required" to do. For standards to be effective, they need to be seen as legitimate by those they may be imposed on.
+
+A mechanism is needed to capture the sense of the community and allow it to progress while recognizing that the community is broad, diffuse, and diverse, and not all stakeholders are present for all conversations.
+
+# PHEP Audience
+<a name="phep-audience"></a>
+The typical audience for PHEPs is the developers and user communities of PyHC packages. Other parts of the PyHC community (hereafter "community") may also use the process (particularly for Informational PHEPs) to document conventions and to manage complex design coordination problems that require collaboration across multiple projects.
+
+This PHEP refers to "PyHC leadership": individuals or groups with statutory authority over the PyHC project. This may be the Principal Investigator or other individuals or groups appropriately designated in the future, such as a steering committee.
+
+# PHEP Types
+<a name="phep-type"></a>
+There are three types of PHEP:
+
+1. A **Standards** Track PHEP describes a new or modified standard for PyHC packages. These standards are requirements that PyHC packages are expected to follow, similar to the original [PyHC standards](https://doi.org/10.5281/zenodo.2529131). Examples include testing requirements, OS support, and dependency handling.
+
+2. An **Informational** PHEP describes a design issue, or provides general guidelines or information to the community, but does not propose a standard. Informational PHEPs do not necessarily represent a community recommendation, so users and implementers are free to ignore Informational PHEPs or follow their advice.
+
+3. A **Process** PHEP describes a process surrounding PyHC, or proposes a change to a process. Process PHEPs are like Standards Track PHEPs but apply to areas other than PyHC packages. Unlike Informational PHEPs, they are more than recommendations, and users are typically not free to ignore them. Examples include procedures, guidelines, and changes to the decision-making process.
+
+# PHEP Applicability
+<a name="phep-applicability"></a>
+**Standards** Track PHEPs are recommended standards for PyHC packages. Most packages should make a good-faith effort to comply. PyHC core packages are strongly recommended to comply and generally must do so, absent a compelling reason not to. A PHEP which core packages cannot agree to implement likely does not have sufficient community support for acceptance. Standards track PHEP compliance must be considered as a factor in evaluating current and potential PyHC packages.
```

Comment:
Intentionally vague so we can change in the future without reopening PHEP 1. I was thinking of two mechanisms, which don't have to be exclusive:

1. We should formally define package review, "core" package, and compliance with standards. This is particularly relevant if there's an option of two-track compliance that is not yet defined. This could set standards like "packages of x type must comply with all Final Standards-track PHEPs within 6 months" (example only, let's not bikeshed if that's a good idea). I'd expect such definition to be in another PHEP, if that's the way we make decisions :) and it's a pretty big chunk of work that puts it out of scope of PHEP 1.
2. I see no reason a PHEP can't set its own scope, including a clause like "this takes effect for core packages six months after Final" or something. We'd have to be careful that package submitters don't need to look at the "package review" PHEP and all other Standards PHEPs to know what they need to comply with, but on the other hand the baseline would be to look at all of them anyhow. The community can discuss these issues on a case-by-case basis for any PHEP that sets its scope.

Reply:
[New language to try and clarify this](https://github.com/heliophysicsPy/standards/blob/68125caadd7c081ecc8fc9f803c6d3ebebebce58/pheps/phep-0001.md?plain=1#L51)

Reply:
Yes, new standards track PHEPs would imply rereview of packages. As I said above, I think it makes more sense for standards enforcement, package "types", review process to be separate from PHEP 1. We may have many processes and they don't all need to be defined in one place.